src/CMakeLists.txt

```diff
@@ -86,13 +84,19 @@ elseif(MACOSX)
 endif()
 
 if(UMF_BUILD_SHARED_LIBRARY)
+    if(WINDOWS AND (NOT UMF_BUILD_OS_MEMORY_PROVIDER))
+        # UMF_BUILD_SHARED_LIBRARY=ON requires UMF_BUILD_OS_MEMORY_PROVIDER=ON on Windows
```

Comment:
This should be fixed by making def and map files generated - we had a JIRA for that, but please insert here a TODO

Reply:
Changed: added a separate def file for that case.